backend/src/Controllers/getElectionRollController.ts

```diff
@@ -0,0 +1,65 @@
+import { ElectionRoll, ElectionRollState } from "../../../domain_model/ElectionRoll";
+import ServiceLocator from "../ServiceLocator";
+import Logger from "../Services/Logging/Logger";
+import { responseErr } from "../Util";
+import { hasPermission, permissions } from '../../../domain_model/permissions';
+
+const ElectionRollModel = ServiceLocator.electionRollDb();
+
+const className = "VoterRolls.Controllers";
+
+const getRollsByElectionID = async (req: any, res: any, next: any) => {
```

Comment:
I think the corresponding function in backend/src/Controllers/voterRolls.controller.ts needs to be deleted?

Reply:
That is still used by the '/Election/:id/finalize' route which gets the rolls and sends the invitations.